HW04/Kate Fedotova/array_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rspec'
+require 'rubocop-rspec'
+require_relative 'arr.rb'
+
+# rubocop:disable Metrics/BlockLength
+describe Array do
+  describe 'my_each' do
+    let(:arr) { [1, 2, 3, 4] }
+
+    it 'has block' do
```

Comment:
Can you add a context? Something like this http://www.betterspecs.org/#contexts
```suggestion
    context 'when has block' do
      it 'returns correct result' do
```